usr/lib/cca_stdll/ccatok.conf

```diff
@@ -0,0 +1,19 @@
+version cca-0
+
+# Optionally specify the expected master key verification patters for the 
+# SYM, AES, and APKA master key. The CCA token does not use the ASYM master key.
```

Comment:
Well are you sure the CCA token does not use the ASYM master key ? Each RSA key pair generate uses it...

Reply:
No APKA is used for 'newer' asymmetric key tokens, and this includes besides ECC also the newer RSA key token types that opencryptoki is using.